public/backend/js/admins.js

```diff
@@ -2,4 +2,21 @@ $(document).ready(function(){
   $('#list_users').DataTable();
   $('#list_books').DataTable();
   $('#list_categories').DataTable();
-});
\ No newline at end of file
+});
+
+function readURL(input) {
+    if (input.files && input.files[0]) {
+        var reader = new FileReader();
+
+        reader.onload = function (e) {
+            $('#image_no').attr('src', e.target.result);
+        }
+        reader.readAsDataURL(input.files[0]);
+    }
+}
+
+$("#image").change(function(){
```

Comment:
[rank C] As I said before, please consider about the way we write an jQuery event.